core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -815,6 +815,12 @@ class AutoTuner(
           s"'$lookup' should be increased since spilling occurred.")
       }
     }
+    // If the user has enabled AQE auto shuffle, the auto-tuner should recommend to disable this
+    // feature before recommending shuffle partitions.
+    val aqeAutoShuffle = getPropertyValue("spark.databricks.adaptive.autoOptimizeShuffle.enabled")
+    if (!aqeAutoShuffle.isEmpty) {
+      appendRecommendation("spark.databricks.adaptive.autoOptimizeShuffle.enabled", "false")
+    }
```

Comment:
Why do we need to check if the property is set or not?
What if we do not explicitly set it to false, the user could mistakenly add it later thinking that it does not harm.
WDYT?

```
appendRecommendation("spark.databricks.adaptive.autoOptimizeShuffle.enabled", "false")
```

Reply:
I see your point on this. This configuration is quite uncommon, and only for AQE. If by default we set it to false, I am not sure if we are adding noise for our users.

Reply:
What I meant is to make this recommendation within the AQE part.
My intuition is that users take the output from the autoTuner and merge it into their job configurations.
It might be useful for users to see that this configuration has to be turned-off when they deploy our recommendation.
Anyway, I am fine with what we have here until @mattahrens maybe decide to have that more informative.